package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wazo-ts-types",
+  "version": "1.0.0",
+  "description": "TS types for Wazo APIs",
+  "main": "index.js",
+  "author": "Wazo (http://wazo.io)",
+  "license": "GPL-3.0-only",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "engines": {
+    "node": "22.x"
```

Comment:
Should we match a node version inside a `.nvmrc` ?

Reply:
yes added here ! https://github.com/TinxHQ/wazo-ts-types/blob/f50e6b2b14a2b212534ff0c7f20798f72f848758/.nvmrc